README.md

```diff
@@ -57,6 +57,13 @@ SSH flags:
 * `-F`, `--ssh-config=FILE`: specify SSH config file used for `ssh -F`
 * `--ssh-persist=(true|false)` (default: `true`): enable ControlPersist
 
+SSHFS flags:
+* `--sshfs-noempty` (default: `false`): enable sshfs nonempty
+
+SFTP server flags:
+* `--driver=DRIVER` (default: `auto`): SFTP server driver. `buitlin` (legacy) or `openssh-sftp-server` (robust and secure, recommended). Automatically chosen by default.
+* `--openssh-sftp-server=BINARY`: OpenSSH SFTP Server binary. Automatically chosen by default
```

Comment:
```suggestion
* `--driver=DRIVER` (default: `auto`): SFTP server driver. `builtin` (legacy) or `openssh-sftp-server` (robust and secure, recommended). Automatically choosen when the binary is detected.
* `--openssh-sftp-server=BINARY`: OpenSSH SFTP Server binary. Automatically detected when installed in default locations.
```

Confirmation for "chosen" vs "choosen" from native speaker welcome, but present tense feels better here.

Reply:
"choosen" doesn't seem correct https://www.google.com/search?q=choosen

Reply:
> "choosen" doesn't seem correct https://www.google.com/search?q=choosen

Yes, I was wrong (that's why I wrote "confirmation welcome", but I guess I should have googled).

I still think the other suggestions are an improvement.

Reply:
updated